rre-dataset-generator/src/test_env/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  solr:
+    image: solr:9.8.1
+    container_name: solr_test
+    ports:
+      - "8983:8983"
+    volumes:
+      - ./volumes/solr:/var/solr
```

Comment:
We don't need a volume here. We don't want to access the Solr index from the local machine

Reply:
Furthermore, if the folder `volumes` is not created and it's going to be created by the Docker daemon with root permissions. Solr does not have the write permission to write into that folder, and it fails.